code/modules/roguetown/roguemachine/bounty.dm

```diff
@@ -320,27 +324,46 @@
 
 	sleep(2 SECONDS)
 
-	if(correct_head)
-		say("A bounty has been sated.")
-		budget2change((reward_amount*2))
+	if(correct_head || grove_marked)
+		if(correct_head)
+			say("A bounty has been sated.")
+			budget2change((reward_amount*2))
+
+		if(grove_marked)
+			say("Nature's mark detected. Issuing summary judgment and commencing sentencing...")
+			M.remove_status_effect(/datum/status_effect/grove_outlaw)
+
+			for(var/obj/structure/grove_wanted/totem in world)
+				totem.marked_individuals -= M.real_name
+				totem.roundstart_marks -= M.real_name
+
+			for(var/mob/living/carbon/human/grove_member in GLOB.player_list)
+				if(grove_member.job in list("Great Druid", "Druid", "Hedge Warden", "Hedge Knight"))
+					to_chat(grove_member, span_green("<b>Grove Judgment Alert:</b> [M.real_name] has been judged and sentenced by the POENA."))
+					playsound(grove_member, 'sound/misc/treefall.ogg', 50, TRUE)
+
+			if(M in SStreasury.bank_accounts)
+				var/money_to_take = round(SStreasury.bank_accounts[M] * 0.5)
```

Comment:
You are already being enslaved. Being collared IS your penance. Forcibly taking any of their stored mammons in addition to that is excessive. If you want to make a monetary penalty, let them work it off by growing sackfuls of grain.